### DISCUSSION TITLE
Phase 3 is sooooooo slow?

### DISCUSSION THREAD

**Discussion by Kali123411** [swar/Swar-Chia-Plot-Manager#857] [Q&A]
Not sure why but phase 3 is taking a really really really long time to complete. My machines resources are barely breaking a sweat so I'm not sure why this is happening...maybe the SSD is about to eat the dust....
![phase3](https://user-images.githubusercontent.com/64617127/119412902-e0168800-bca9-11eb-81b2-a01840da635e.PNG)

**Comment by tracy32792** [comment]
I need more than eight hours in phase three, and it's unclear exactly what the problem is

**Reply by Kali123411** [reply]
are you also using a Seagate SSD ? I've noticed my Samsung EVO drives perform way better.
